page_objects/Base_Page.py

```diff
@@ -607,6 +657,8 @@ def log_result(self,flag,positive,negative,level='info'):
             self.success(positive,level=level)
         if flag is False:
             self.failure(negative,level=level)
+        if flag is None:
```

Comment:
The result flag can be either success/failure.
We may have to make the change at the place where it takes None.